legacy/gcc_9/Dockerfile

```diff
@@ -24,11 +23,10 @@ RUN dpkg --add-architecture i386 \
        binutils \
        wget \
        git \
-       libc6-dev-i386 \
        libc6-dev \
-       linux-libc-dev:i386 \
-       g++-9-multilib \
```

Comment:
multilib installs both x86 and x64 libraries. We no longer build 32-bits in CCI.